efficientdet/keras/eval.py

```diff
@@ -26,16 +27,36 @@
 from keras import efficientdet_keras
 from keras import postprocess
 
+flags.DEFINE_string('validation_file_pattern', None,
+                    'Glob for evaluation tfrecords (e.g., COCO val2017 set)')
+flags.DEFINE_string(
+    'val_json_file', None,
+    'COCO validation JSON containing golden bounding boxes.')
+flags.DEFINE_string('model_name', 'efficientdet-d0',
+                    'Model name: the efficientdet model to use.')
+flags.DEFINE_string('checkpoint', None, 'Location of the checkpoint to evaluate.')
+
+FLAGS = flags.FLAGS
+
 def main(_):
-  config = hparams_config.get_efficientdet_config('efficientdet-d0')
+  if FLAGS.validation_file_pattern is None:
```

Comment:
You can use flags.mark_flag_as_required('validation_file_pattern')
see: https://abseil.io/docs/python/guides/flags

Reply:
Done.